charts/backbeat-api/templates/_helpers.tpl

```diff
@@ -0,0 +1,32 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "backbeat-api.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
```

Comment:
As commented below, I would think it's better to fail early if names are > 64 chars to avoid any further hard to diagnose issues, but it's good if we can have this check done around here anyway (but failing instead of truncating).

Reply:
No, this is how it's done in every chart (not just ours) out there.